components/viaduct/src/lib.rs

```diff
@@ -19,10 +19,7 @@ pub use headers::{consts as header_names, Header, HeaderName, Headers, InvalidHe
 pub use settings::GLOBAL_SETTINGS;
 
 pub(crate) mod msg_types {
-    include!(concat!(
-        env!("OUT_DIR"),
-        "/mozilla.appservices.httpconfig.protobuf.rs"
-    ));
+    include!("mozilla.appservices.httpconfig.protobuf.rs");
```

Comment:
any reason not to do these as e.g. `#[path = "mozilla.appservices.httpconfig.protobuf.rs"] pub(crate) mod msg_types;`?

Reply:
Had no idea that was a thing!

Reply:
Well the problem is that these files then get formatted by `cargo fmt`, and the `ignore` directive is only on the nightly channel... 😐